tests/utils.go

```diff
@@ -529,8 +529,9 @@ func NewRandomVMWithDataVolumeAndUserDataInStorageClass(imageUrl, namespace, use
 //
 // Deprecated: Use libvmi directly
 func NewRandomVMIWithEphemeralDiskAndConfigDriveUserdataHighMemory(containerImage string, userData string) *v1.VirtualMachineInstance {
-	vmi := NewRandomVMIWithEphemeralDiskAndConfigDriveUserdata(containerImage, userData)
-
+	vmi := libvmi.NewCirros(
+		libvmi.WithCloudInitConfigDriveUserData(userData),
+	)
 	vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("512M")
```

Comment:
please, use `libvmi.WithResourceMemory("512M")`
or
directly address `NewRandomVMIWithEphemeralDiskAndConfigDriveUserdataHighMemory` in this PR (just one usage)

Reply:
I will go with option 1 because we plan to remove `NewRandomVMIWithEphemeralDiskAndConfigDriveUserdataHighMemory` as well.
Thanks for looking at this :slightly_smiling_face: